radicle-node/src/test/peer.rs

```diff
@@ -319,10 +324,9 @@ where
         msgs.into_iter()
     }
 
-    /// Get a draining iterator over the peer's emitted events.
-    pub fn events(&mut self) -> impl Iterator<Item = Event> + '_ {
-        self.outbox()
-            .filter_map(|io| if let Io::Event(e) = io { Some(e) } else { None })
+    /// Get a stream of the peer's emitted events.
+    pub fn events(&mut self) -> chan::Receiver<Event> {
```

Comment:
maybe I should rename this to `subscribe` too?

also what should I do with the `fn` after this, `pub fn outbox`? is that still relevant?

Reply:
I'd actually rename the other `Handle::subscribe` function to `Handle::events`, because it's clearer that you're getting events.

`outbox` is still useful as it's the `Io` actions that are returned from there.

Reply:
I renamed all `subscribe`s to `event`, [see](https://github.com/radicle-dev/heartwood/compare/096b533df2b3e727e32ed084c3f5e301bafd5a6d..350133ebabb47b1ed79128eb691c771235b1253f#diff-85df6dd835d2ce99d0219ad5e696f3c00dc35d94ccf1876ec98d3da15a36883fR89)